bindings/python/module.cpp

```diff
@@ -72,4 +77,36 @@ PYBIND11_MODULE(_pyghex, m)
     pyghex::unstructured::register_field_descriptor(m);
     pyghex::unstructured::register_pattern(m);
     pyghex::unstructured::register_communication_object(m);
+
+    m.def("expose_cpp_ptr", [](pyghex::context_shim* obj) {return reinterpret_cast<uintptr_t>(&obj->m);} );
```

Comment:
please `#include <cstdint>` and
```suggestion
    m.def("expose_cpp_ptr", [](pyghex::context_shim* obj) {return reinterpret_cast<std::uintptr_t>(&obj->m);} );
```